Mapify/SceneInitializers/GameContent/WorldMapSetup.cs

```diff
@@ -16,6 +16,7 @@ public class WorldMapSetup : SceneSetup
     {
         private static bool modifiedMaterial;
         private static Texture defaultTexture;
+        public static bool showStationNamesOnMap = false;
```

Comment:
Nit: No need to initialize a boolean to false, that's implicit.

Reply:
I like to be explicit, makes the code easier to read. Also, I forgot that bools are false by default in C# :)

Ill change it